src/main/java/step3RacingCarCompetition/RacingCarInfo.java

```diff
@@ -0,0 +1,22 @@
+package step3RacingCarCompetition;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class RacingCarInfo {
+
+    private ArrayList<String> racingCarMovingData;
```

Comment:
변수의 타입은 추상화된 타입으로 정의해 주세요
```suggestion
    private List<String> racingCarMovingData;
```

Reply:
감사합니다. 추상화하여 수정했습니다.